src/helpers/notifier/index.js

```diff
@@ -0,0 +1,52 @@
+/* global Notification */
+const invariant = require('invariant')
+const { MERGE, CONVERT } = require('../constants')
+const text = require('./text.json')
+
+let imgPath
+
+module.exports = {
+  init,
+  notify
+}
+
+function init (appPath) {
+  imgPath = `${appPath}/img`
+}
+
+function notify ({ didSucceed, operation }) {
+  invariant([MERGE, CONVERT].includes(operation), 'Invalid notification operation')
+
+  const { title, body, icon } = genInfo(didSucceed, operation)
+
+  const n = new Notification(title, {
+    body,
+    icon
+  })
+
+  return n
+}
+
+function genInfo (didSucceed, operation) {
+  const index = getIndex()
+  const operationText = operation === MERGE ? 'Merge' : 'Conversion'
+
+  if (didSucceed) {
+    return {
+      title: `${operationText} complete!`,
+      body: text.success[index],
+      icon: `${imgPath}/success.png`
+    }
+  }
+
+  return {
+    title: `Oh no! ${operationText} failed.`,
+    body: text.failure[index],
+    icon: `${imgPath}/failure.png`
+  }
+}
+
+function getIndex () {
+  const rnd = Math.random()
+  return Math.floor(rnd * text.messageCount)
```

Comment:
Can you use `array.length` please 😄

Reply:
Thought this might be more explicit in communicating that `success` and `failure` must have the same length since `getIndex` doesn't know which one the message is being drawn from 😄

Reply:
Or you could have written a test that ensures they do